### PR TITLE
[Graylog_sidecar] Init Role

### PR DESCRIPTION
**Pull request by greedybro**

### CODE REVIEW

File: manala.graylog_sidecar/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+
+# Install
+manala_graylog_sidecar_install_version: ~
+manala_graylog_sidecar_install_packages:
+  - https://github.com/Graylog2/collector-sidecar/releases/download/{{ manala_graylog_sidecar_install_version }}/graylog-sidecar_{{ manala_graylog_sidecar_install_version }}-1_amd64.deb
```
Comment (hyvs) [inline manala.graylog_sidecar/defaults/main.yml]: Should we have an `manala_graylog_sidecar_install_default_package` instead ? I am afraid that this URL can change in near future

File: manala.graylog_sidecar/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+# Config
+manala_graylog_sidecar_config_file: /etc/graylog/sidecar/sidecar.yml
+manala_grafana_config_template:     ~
+manala_graylog_sidecar_config:      []
+
+# Services
+manala_sensu_go_services:          []
```
Comment (hyvs) [inline manala.graylog_sidecar/defaults/main.yml]: typo

File: manala.graylog_sidecar/lookup_plugins/manala_graylog_sidecar_deb.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
+from __future__ import (absolute_import, division, print_function)
```
Comment (hyvs) [inline manala.graylog_sidecar/lookup_plugins/manala_graylog_sidecar_deb.py]: You must gitignore `*.pyc` file.

File: manala.graylog_sidecar/tasks/install.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+
+- name: install > Deb
+  apt:
+    deb: "{{ item }}"
+  with_items: "{{ lookup(
+      'manala_graylog_sidecar_deb',
```
Comment (hyvs) [inline manala.graylog_sidecar/tasks/install.yml]: Why this lookup ? We handle only *.deb in our install tasks. 

cc @nervo

Comment (greedybro) [reply manala.graylog_sidecar/tasks/install.yml]: revert to not using lookup

**Pull request merged**